sphinxlint/__main__.py

```diff
@@ -109,6 +113,15 @@ def __call__(self, parser, namespace, values, option_string=None):
         help="comma-separated list of fields used to sort errors by. Available "
         f"fields are: {SortField.as_supported_options()}",
     )
+    parser.add_argument(
+        "-p",
+        "--processes",
+        action=StoreNumProcsAction,
+        type=int,
+        help="Number of processes to use, defaults to using os.cpu_count(). "
+        "Values <= 1 are all considered == 1",
+        default=None
+    )
```

Comment:
I suggest following a similar pattern to `sphinx-build`, at least use the same `-j` and `--jobs` names:

```
  -j N, --jobs N    build in parallel with N processes where possible 
                    (special value "auto" will set N to cpu-count)
```

Reply:
Done!